### ISSUE TITLE
"Manage My Group" button appears when it shouldn't

### ISSUE THREAD

**Issue by mscholtz** [washingtontrails/vms#127]
I'm getting it on every WP when logged in as Admin, whether or not I'm registered for that WP, and even when there are no Groups for that WP.

Screenshot example:

![image](https://cloud.githubusercontent.com/assets/931031/7973740/d022d0a2-0a0d-11e5-9dde-8fad9af44c2c.png)

**Comment by davisagli** [comment]
Fixed (it will still show up for admins, but only on work parties that have groups).

**Comment by mscholtz** [comment]
If I'm an admin not part of a group, and there's multiple groups, what
happens when I click it?

On Wed, Jun 3, 2015 at 5:02 PM, David Glick <notifications@github.com>
wrote:

> Fixed (it will still show up for admins, but only on work parties that
> have groups).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/washingtontrails/vms/issues/127#issuecomment-108651824>
> .
>



-- 

Matthew Scholtz
Percolator Consulting
(206) 289-0816 x3

http://percolatorconsulting.com/

**Comment by davisagli** [comment]
I meant for it to just use the first group for now, but I don't think that's working yet. Ideally it should let the admin choose which group they want to add a person to.

**Comment by mscholtz** [comment]
Do we need this?  I would suggest that the right way for an admin to add
someone to a group is via SF.

On Wed, Jun 3, 2015 at 5:17 PM, David Glick <notifications@github.com>
wrote:

> I meant for it to just use the first group for now, but I don't think
> that's working yet. Ideally it should let the admin choose which group they
> want to add a person to.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/washingtontrails/vms/issues/127#issuecomment-108654787>
> .
>



-- 

Matthew Scholtz
Percolator Consulting
(206) 289-0816 x3

http://percolatorconsulting.com/

**Comment by davisagli** [comment]
I'm of the general opinion that if an Admin happens to be logged in to VMS, they should be able to do anything a crew leader can do.

**Comment by mscholtz** [comment]
Can crew leaders add volunteers to groups?

On Wed, Jun 3, 2015 at 5:45 PM, David Glick <notifications@github.com>
wrote:

> I'm of the general opinion that if an Admin happens to be logged in to
> VMS, they should be able to do anything a crew leader can do.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/washingtontrails/vms/issues/127#issuecomment-108659228>
> .
>



-- 

Matthew Scholtz
Percolator Consulting
(206) 289-0816 x3

http://percolatorconsulting.com/

**Comment by davisagli** [comment]
Current status is that I meant for them to be able to but there's a bug that makes it not show up. It was never really specified though.

**Comment by mscholtz** [comment]
Yeah, I don't recall hearing that this is something that CLs should be able to do in general.
@WaBirder57 or @kuffelman what say you?

**Comment by davisagli** [comment]
I just remembered that when I talked to Darrell about the UI for adding people to a work party, it sounded like he was thinking of it as something that would be available to crew leaders, not just group organizers. It's implied by the "Add Member" button on the wireframes page 24. Of course WTA has the final say.

**Comment by WaBirder57** [comment]
Crew Leaders will not be adding (signing up) Group Members to a work party.  Just the Group Coordinator and WTA Staff.

**Comment by mscholtz** [comment]
@davisagli just happened upon this.  Did this ever get resolved after Charlie's last comment?

**Comment by davisagli** [comment]
Not sure. I haven't generally been looking at tickets tagged as "Reviewing" so it may have gotten overlooked.
